kolena/dataset/evaluation.py

```diff
@@ -227,6 +242,7 @@ def upload_results(
     dataset: str,
     model: str,
     results: Union[DataFrame, List[Tuple[EvalConfig, DataFrame]]],
+    thresholded_fields: Optional[List[str]] = None,
```

Comment:
Haven't been paying the closest attention here, but is there any reason we can't infer that a field is thresholded based on the dataframe? Having another field here increases the complexity of our API.

Reply:
1. There could be multiple fields that store thresholded metrics kind of data.
2. For users using thresholded metrics, they are already in the "more advanced" scenario who have to have the knowledge, thus specifying the fields is comparatively not much extra burden.
3. Personally, I prefer to avoid inferring based on naming convention.

However, we do face the same issue regardless when it comes to importing through UI. Currently thinking to support configuration with this infer from UI side.

Reply:
Synced with @ly-kolena and Mehrad offline -- being explicit with the name in the public client, and having the web app do the "smart inference" seems like a fine approach. No change needed here.